x/auth/vesting/module.go

```diff
@@ -94,7 +94,7 @@ func (AppModule) RegisterInvariants(_ sdk.InvariantRegistry) {}
 
 // Route returns the module's message router and handler.
 func (am AppModule) Route() sdk.Route {
-	return sdk.NewRoute(types.RouterKey, NewHandler(am.accountKeeper, am.bankKeeper))
+	return sdk.Route{}
```

Comment:
Yeah returning just `sdk.Route{}` seems like it could lead to errors possibly? I'd recommend doing anything we can to encourage developers to not use deprecated functionality and panicing seems OK to me.

Reply:
Adding panic is breaking [RegisterRoutes](https://github.com/cosmos/cosmos-sdk/blob/master/types/module/module.go#L282-L284). We are not registering empty [router](https://github.com/cosmos/cosmos-sdk/blob/master/types/module/module.go#L282), I think returning `sdk.Route{}` is fine.